File: corpus/py/D;0123.py
int n,k,i
print("Insira o tamanho do vetor: \n")
n=int(input())
for i=0 in range(0, n):
    print("Insira o "+(i +1)+"º Valor do vetor. \n")
comparaVetor(v,n,k)
def comparaVetor(int v[],int n, int k):
    int i
    for i=0 in range(0, n):
        if v[i]>20:

File: corpus/2018q1/_p2q1_ok/_check/D;0123.java

import java.util.Scanner;

public class Questao1 {

    public static void main(String[] args) {

       int n,k,i;
       Scanner sc= new Scanner(System.in);

       System.out.println("Insira o tamanho do vetor: \n");
       n=sc.nextInt();
        int v[]= new int[n];
       for(i=0;i<n;i++){
           System.out.println("Insira o "+(i+1)+"º Valor do vetor. \n");
           v[i]=sc.nextInt();   
       }

        comparaVetor(v,n,k);
    }
    public static boolean comparaVetor(int v[],int n, int k){
      int i;

                for(i=0;i<n;i++){
                   if (v[i]>20)){

                }  
              }

    }
}
